src/components/Comments.jsx

```diff
@@ -41,11 +41,32 @@ function Comments(props) {
         let newComment = result.data.comment[0];
         setLoaded(true);
         setComments((prev) => [newComment, ...prev]);
-        setSuccessMessage(
+        setSuccessPostMessage(
           "Thanks for your comments - your post has been added!"
         );
         setTimeout(() => {
-          setSuccessMessage(null);
+          setSuccessPostMessage(null);
+        }, 3000);
+      })
+      .catch((error) => {
+        setError(error);
+      });
+  };
+
+  //deleting a comment
+  const handleDelete = (event, comment_id) => {
+  
+    setComments((prev) => {
+      return prev.filter((comment) => comment.comment_id !== comment_id);
+    });
+    let url = deleteCommentUrl(comment_id);
+    axios
+      .delete(url)
+      .then((result) => {
+        setLoaded(true);
+        setDeleteMessage("Your comment has been deleted!");
```

Comment:
You have set your delete message here, could you use this delete message on line 86 instead of re-writing it?